src/index.html

```diff
@@ -1,18 +1,44 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
-    <title>Calendar</title>
-    <link
-      rel="stylesheet"
-      href="styles/index.scss"
-    />
-  </head>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Calendar</title>
+  <link rel="stylesheet" href="./styles/index.scss" />
```

Comment:
The link tag is referencing an SCSS file (`index.scss`). Browsers cannot directly use SCSS files; they need CSS files. Please ensure that your SCSS is compiled to CSS and update the link to point to the compiled CSS file.